Mathlib/AlgebraicTopology/SingularSet.lean

```diff
@@ -0,0 +1,71 @@
+/-
+Copyright (c) 2024 Johan Commelin. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Johan Commelin, Scott Morrison, Adam Topaz
```

Comment:
Can you make sure the year is correct?

Reply:
Shouldn't it be the year that this file was created?

Reply:
Ah, I guess [we've argued this before](https://leanprover.zulipchat.com/#narrow/stream/113488-general/topic/Copyright.20header/near/280420005). I always take it to be the year the oldest definition/lemma of the file was created.